draft-ietf-ppm-dap.md

```diff
@@ -4936,4 +4961,56 @@ Each of these requires registration of a codepoint or other value; see
   impacts the security of DAP with respect to the threat model in
   {{sec-considerations}}.
 
+* When a document defines a new VDAF ({{!VDAF, Section 5}}), it must specify an
+  encoding of that VDAF's configuration like the ones in
+  {{vdaf-configuration-encodings}}.
+
 --- back
+
+# VDAF Configuration Encodings
+
+This section provides encodings of configuration of the Prio3 family of VDAFs.
+These are included in `InputShareAad` structures ({{upload-request}}) to prevent
+cross-VDAF confusion attacks.
+
+## Prio3Count
+
+There are no parameters for this VDAF so `Empty` ({{basic-definitions}}) is
+used.
+
+## Prio3Sum
+
+~~~ tls-presentation
+struct {
+    uint32 max_measurement; /* largest summand */
+} Prio3SumConfig;
+~~~
+
+## Prio3SumVec
+
+~~~ tls-presentation
+struct {
+    uint32 length;       /* length of the vector */
+    uint8 bits;          /* bit length of each summand */
+    uint32 chunk_length; /* size of each proof chunk */
+} Prio3SumVecConfig;
+~~~
+
+## Prio3Histogram
+
+~~~ tls-presentation
+struct {
+    uint32 length;       /* number of buckets */
+    uint32 chunk_length; /* size of each proof chunk */
+} Prio3HistogramConfig;
+~~~
+
+## Prio3MultihotCountVec
+
+~~~ tls-presentation
+struct {
+    uint32 length;       /* length of the vector */
+    uint32 chunk_length; /* size of each proof chunk */
+    uint32 max_weight;   /* largest vector weight /
+} Prio3MultihotCountVecConfig;
+~~~
```

Comment:
I didn't include Poplar1 here because DAP isn't capable of running Poplar1 or any VDAF that collects batches multiple times.

Reply:
Poplar1 can be used in "attributed-based metrics" mode, providing a limited version of Mastic's capability. I think it should be included.